_docs/protocols/2017-01-06-binary.md

```diff
@@ -215,7 +215,17 @@ ghci> hexEncode $ Data.HashMap.Strict.fromList [(1 :: Word8, 127 :: Word64), (2,
 newtype MessageName = MessageName BS.ByteString
 ~~~
 
-`MessageName` is encoded in same manner as `ByteString`.
+`MessageName` is encoded in same manner as `ByteString`. Note that encoded `ByteString` is itself encoded `UnsignedVarInt`, or concatination of two encoded
```

Comment:
I mean that it's probably better to rephrase. Like, `MessageName` is represented as `UVarInt` which is encoded as `ByteString` after that prefixed by bytes length.